docs/AdvancedUsage.md

```diff
@@ -0,0 +1,172 @@
+| **ttProvideLogs**     | **timeout**: int - Defines the maximum execution time for providing all ecu test logs. Set to 0 to disable timeout.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | /                                            |
+| **ttRunPackage**      | **testCasePath**: String - The path to the test case file that should be started. A test case file can be a package, project, project archive or analysis job. The path must either be an absolute path or a relative path to the Packages directory in the workspace. <br/><br/> **testConfig**: [TestConfig](#testconfig) - Contains settings for the ecu test configuration. <br/><br/> **executionConfig**: [ExecutionConfig](#executionconfig) - Contains settings to handle ecu test executions  <br/><br/> **packageConfig**: [PackageConfig](#packageconfig) - Contains package parameters <br/><br/> **analysisConfig**: [AnalysisConfig](#analysisconfig) - Contains settings for analysis execution.                                                                                                                    | [TestResult](#testresult)                    |
+| **ttRunProject**      | **testCasePath**: String - The path to the test case file that should be started. A test case file can be a package, project, project archive or analysis job. The path must either be an absolute path or a relative path to the Packages directory in the workspace. <br/><br/> **testConfig**: [TestConfig](#testconfig) - Contains settings for the ecu test configuration. <br/><br/> **executionConfig**: [ExecutionConfig](#executionconfig) - Contains settings to handle ecu test executions                                                                                                                                                                                                                                                                                                                              | [TestResult](#testresult)                    |
```

Comment:
run package should reference packages for the testCasePath and run project should reference projects